meshPointExample.py

```diff
@@ -52,12 +58,26 @@
                 # trimesh expects XYZ points, not ZYX, so we'll swap these axes
                 xyz_point = [point[2], point[1], point[0]] 
 
+        
                 # check to see whether the point is contained within the current mesh
                 if mesh.contains([xyz_point]):
+                    filename = 'C:/Users/emmak/Desktop/' + mesh_name + '.csv'
+                    if os.path.exists(filename):
+                        f = open(filename, 'a', newline = '')

```

Comment:
In the case that there is an existing CSV file, I think it's better to overwrite it than to append to it. If we append, one might accidentally run this script twice and misinterpret the resulting CSV to mean that there are twice as many points within the surface.